eng/CodeChecks.ps1

```diff
@@ -71,7 +76,7 @@ try {
     if ($LastExitCode -ne 0) {
```

Comment:
I see this is where you are failing if there are api changes. I'm OK with this approach.